app/views/curation_concerns/file_sets/upload/_form_fields.html.erb

```diff
@@ -12,14 +12,14 @@
                 <span class="btn btn-primary fileinput-button">
                     <i class="glyphicon glyphicon-plus" aria-hidden="true"></i>
                     <span aria-hidden="true">Select files...</span>
-                    <input type="file" name="files[]" multiple />
```

Comment:
@jcoyne hmmm. it's blowing up here:
https://github.com/projecthydra-labs/curation_concerns/blob/master/app/views/curation_concerns/file_sets/upload/_form_fields.html.erb#L4
with `undefined local variable or method `upload_set_id'`